package.json

```diff
@@ -17,6 +17,9 @@
   },
   "homepage": "https://github.com/NiiazalyDzhumaliev/weatherApp#readme",
   "devDependencies": {
+    "css-loader": "^5.0.0",
+    "file-loader": "^6.1.1",
+    "style-loader": "^2.0.0",
     "webpack": "^5.1.3",
     "webpack-cli": "^4.1.0"
   }
```

Comment:
## Webpack configuration

- [ ] All webpack related packages should be included in `package.json` like `webpack`, `webpack-cli`, `webpack-dev-server` etc. Also, those packages should be added as **devDependencies.** We are currently missing some of these in this file, let's add them. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/devdep_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/))

- [ ] We need to have necessary run common commands like `build`, `development` to be present in `package.json` inside \`scripts'. Here is a basic example we can draw inspiration from 👇 ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/scripts_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/))

```javascript
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```